docs/aspnetcore-mvc.md

```diff
@@ -321,11 +325,8 @@ The `OktaMvcOptions` class configures the Okta middleware. You can see all the a
 | Scope                     | No           | The OAuth 2.0/OpenID Connect scopes to request when logging in. The default value is `openid profile`. |
 | GetClaimsFromUserInfoEndpoint | No       | Whether to retrieve additional claims from the UserInfo endpoint after login. The default value is `true`. |
 | ClockSkew                 | No           | The clock skew allowed when validating tokens. The default value is 2 minutes. |
-| OnTokenValidated                 | No           | The event invoked after the security token has passed validation and a ClaimsIdentity has been generated. |
-| OnUserInformationReceived                 | No           | The event invoked when user information is retrieved from the UserInfoEndpoint. The `GetClaimsFromUserInfoEndpoint` value must be `true` when using this event. |
-| OnOktaApiFailure          | No           | The event invoked when a failure occurs within the Okta API. |
-| OnAuthenticationFailed    | No           | The event invoked if exceptions are thrown during request processing. |
 | Proxy                     | No           | An object describing proxy server configuration.  Properties are `Host`, `Port`, `Username` and `Password` |
+|OpenIdConnectEvents | No |  Specifies the [events](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.remoteauthenticationevents.onremotefailure) which the underlying OpenIdConnectHandler invokes to enable developer control over the authentication process.|
```

Comment:
This link to the MS docs looks not correct. Should be probably this https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.openidconnect.openidconnectevents

Reply:
good catch 👍